src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -319,6 +319,34 @@ valid_file_mem_backends = @DEFVALIDFILEMEMBACKENDS@
 # Default 0-sized value means unlimited rate.
 #tx_rate_limiter_max_rate = 0
 
+# pvpanic device is a simulated ISA device, through which a guest panic
+# event is sent to qemu, and a QMP GUEST_PANICKED event is generated.
+# This allows management apps to be notified and respond to the event.
+#
+# Note:
+#   To enable pvpanic device, you must set
+#   CONFIG_PVPANIC=y for your guest kernel.
```

Comment:
Out of interest, what happens if the guest kernel doesn't include `CONFIG_PVPANIC`? Would it just be a "NOP" if all the new options were enabled in the config file and a panic occured?

Reply:
If without `CONFIG_PVPANIC` in guest kernel, QEMU will not send `GUEST_PANICKED`  event, and we can't get a chance to trigger operation of dump guest memory. Other options enabled in config file will have no effects.